vendor/bat-native-ledger/src/bat/ledger/internal/media/media.cc

```diff
@@ -54,10 +54,13 @@ std::string Media::GetLinkType(
   return type;
 }
 
-void Media::ProcessMedia(const std::map<std::string, std::string>& parts,
-                               const std::string& type,
-                               ledger::VisitDataPtr visit_data) {
-  if (parts.size() == 0 || !ledger_->GetRewardsMainEnabled() || !visit_data) {
+void Media::ProcessMedia(
+    const std::map<std::string, std::string>& parts,
+    const std::string& type,
+    ledger::VisitDataPtr visit_data) {
+  if (parts.size() == 0 ||
```

Comment:
Checking for `parts.empty()` is more concise.

Reply:
done